scripts/trustchain_crawler_plugin.py

```diff
@@ -6,17 +6,14 @@
 import os
 import signal
 import sys
-from asyncio import all_tasks, coroutine, ensure_future, gather, get_event_loop, sleep
-
-sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+from asyncio import all_tasks, ensure_future, gather, get_event_loop, sleep
 
 from ipv8.REST.rest_manager import RESTManager
 from ipv8.attestation.trustchain.settings import TrustChainSettings
 
-import yappi
-
 from ipv8_service import IPv8
 
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```

Comment:
This is wrong.

The path modification to allow imports from `ipv8.*` should be before the imports from `ipv8.*`.

Reply:
I changed the order because I saw it was a flake8 violation. I will change it back, however 👍